### ISSUE TITLE
`Chrome 128` seems to have problem rendering WebGL traces including `parcoords`, `mesh3d`, `surface`

### ISSUE THREAD

**Issue by archmoj** [plotly/plotly.js#7130]
As discovered in #7126 this might be either a regression in Chrome 128 or a bug in plotly.js & stackgl dependencies.
cc: @birkskyum 

Error message: 
```
TypeError: Failed to execute 'uniformMatrix4fv' on 'WebGLRenderingContext': The provided float value is non-finite.
```

**Comment by archmoj** [comment]
Keeping an eye on https://issues.chromium.org/issues?q=componentid:1456284.

**Comment by archmoj** [comment]
I could confirm that e.g. [this parcoords graph](https://codepen.io/MojtabaSamimi/pen/oNryozX?editors=0010) is not rendering on Chrome 128 as well as Edge 128.

**Comment by archmoj** [comment]
Other applications also encountered this:
https://community.esri.com/t5/arcgis-online-questions/map-error-in-new-chrome-update-128-0-6613-85/td-p/1525829

**Comment by archmoj** [comment]
The regression is now fixed by Chromium 128.0.6613.113
https://chromium.googlesource.com/chromium/src/+/refs/tags/128.0.6613.113
Please consider updating your browser if you encountered this bug.
Closing.

**Comment by percybolmer** [comment]
Had this error occur, patch worked like a charm! 
Thanks for updating regarding this here.
